paddle/fluid/operators/lod_tensor_to_array_op.cc

```diff
@@ -131,9 +131,7 @@ class LoDTensorToArrayOp : public framework::OperatorBase {
       }
     }
 
-    auto &outputs = *const_cast<framework::Scope &>(scope)
-                         .Var()
-                         ->GetMutable<std::map<size_t, framework::Tensor>>();
+    std::map<size_t, framework::Tensor> outputs;
```

Comment:
The tensor of outputs is shared from `out[i]` by calling `out[i].Slice`, it doesn't matter that place in stack space.